src/common/internal-plugins.js

```diff
@@ -38,10 +38,16 @@ module.exports = [
         return eval("require")("../language-js/parser-typescript").parsers
           .typescript;
       },
+      /**
+       * TODO: Remove this old alias in a major version
+       */
       get "typescript-eslint"() {
-        return eval("require")("../language-js/parser-typescript").parsers[
-          "typescript-eslint"
-        ];
+        return eval("require")("../language-js/parser-typescript").parsers
+          .typescript;
+      },
+      get "typescript-estree"() {
+        return eval("require")("../language-js/parser-typescript").parsers
+          .typescript;
```

Comment:
I don't know if we should add a new alias

Reply:
Removed 👍